eZ/Publish/Core/Persistence/Legacy/Tests/Notification/Gateway/DoctrineDatabaseTest.php

```diff
@@ -51,7 +51,7 @@ public function testInsert()
         $data = $this->loadNotification($id);
 
         $this->assertEquals([
-            'id' => "$id",
+            'id' => (string)$id,
```

Comment:
Actually no casting needed, it should be an `int` which is already guaranteed by the above `insert()` call. You can also get rid of an extra `" "`

Reply:
Ok, will do.